git.sh

```diff
@@ -8,3 +8,6 @@ git reset --hard origin/main
 
 # Force pull
 git pull origin main --force
+
+# Install dependencies
+npm i
```

Comment:
```suggestion
npm ci
```

This won't try and update anything and will install only the package-lock.json checkins.

Otherwise 👍🏻  I hope this works!